Neos.Flow/Classes/Http/Component/ComponentContext.php

```diff
@@ -12,7 +12,7 @@
  */
 
 use Neos\Flow\Annotations as Flow;
-use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ServerRequestInterface;
```

Comment:
Need an `ServerRequestInterface` here because a couple of places depend on the request from the Context providing `getAttribute(...)` in order to retrieve e.g. the baseUri